lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -414,15 +435,39 @@ def add_app_targets
                 end
                 # For macOS we do not code sign the appbundle because we do not code sign the frameworks either.
                 configuration.build_settings['CODE_SIGN_IDENTITY'] = '' if target.platform == :osx
+
+                target.build_settings_for_spec(app_spec).merged_pod_target_xcconfigs.each_key do |setting|
+                  configuration.build_settings.delete(setting)
+                end
               end
 
               create_app_target_embed_frameworks_script(app_spec)
               create_app_target_copy_resources_script(app_spec)
+              add_resources_to_target(target.file_accessors.find { |fa| fa.spec == app_spec }.resources, app_native_target)
```

Comment:
wait this was missing? Isn't there another method above that installs all of these?

Reply:
it was being done in create_app_target_copy_resources_script before